hypervisor/arch/x86/guest/instr_emul.c

```diff
@@ -1589,8 +1591,8 @@ vie_size2mask(uint8_t size)
 
 int
 vie_calculate_gla(enum vm_cpu_mode cpu_mode, enum cpu_reg_name seg,
-	struct seg_desc *desc, uint64_t offset, uint8_t length, uint8_t addrsize,
-	uint32_t prot, uint64_t *gla)
+	struct seg_desc *desc, uint64_t offset, uint8_t length,
+	uint8_t addrsize, uint32_t prot, uint64_t *gla)
 {
```

Comment:
format change only?

Reply:
Yes, what I changed here is to make sure that the line length is less than 80.